Quotient/e2ee/cryptoutils.cpp

```diff
@@ -320,3 +320,20 @@ std::vector<byte_t> Quotient::base58Decode(const QByteArray& encoded)
     std::reverse(result.begin(), result.end());
     return result;
 }
+
+QUOTIENT_API QByteArray Quotient::sign(const QByteArray& key, const QByteArray& data)
```

Comment:
```suggestion
QByteArray Quotient::sign(const QByteArray& key, const QByteArray& data)
```
It's enough to say `QUOTIENT_API` in the header file.